src/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -64,7 +57,7 @@ const GuaranteeBanner: React.FC<{ conversationID: string }> = ({
 
   return showBanner ? (
     <Banner variant="brand">
-      <GuaranteeIcon mr={1} fill="white100" />
+      <VerifiedIcon mr={1} fill="white100" />
       To be covered by the Artsy Guarantee, always communicate and pay through
       the Artsy platform.
     </Banner>
```

Comment:
Wound up just no-verifying to get the change into this file. Bunch of effect dependencies errors in here.